src/modules/lua.c

```diff
@@ -73,11 +73,17 @@ mtev_lua_lmc_alloc(mtev_dso_generic_t *self, mtev_lua_resume_t resume) {
 void
 mtev_lua_lmc_free(lua_module_closure_t *lmc) {
   if(lmc) {
+    if(lmc->lua_state) lua_close(lmc->lua_state);
```

Comment:
there are a couple of functions that peek into the lua_state via `mtev_lua_states`: `mtev_console_lua_thread_reporter_json`, `mtev_console_lua_thread_reporter_ncct`. I was worried they'd try looking into a `lua_state` that got closed.